.pre-commit-config.yaml

```diff
@@ -29,30 +41,46 @@ repos:
         exclude: ^(conda\.)?recipe/meta.yaml
       # catch git merge/rebase problems
       - id: check-merge-conflict
+  - repo: https://github.com/python-jsonschema/check-jsonschema
+    rev: 0.22.0
+    hooks:
+      # verify GitHub configs
+      - id: check-github-actions
+      - id: check-github-workflows
+  # Python verification and formatting
+  - repo: https://github.com/Lucas-C/pre-commit-hooks
+    rev: v1.4.2
+    hooks:
+      # auto inject license blurb
+      - id: insert-license
+        files: \.py$
+        args: [--license-filepath, .github/disclaimer.txt, --no-extra-eol]
+        exclude: ^conda/auxlib/
   - repo: https://github.com/asottile/pyupgrade
     rev: v3.3.1
     hooks:
+      # upgrade standard Python codes
       - id: pyupgrade
-        args: ["--py37-plus"]
+        args: [--py38-plus]
```

Comment:
Harsh to switch back to an unquoted "reader has to remember yaml's string rules" string?

Reply:
oh, my intention is to enable an auto formatter for YAMLs in a separate PR, it does this for us